web_editor/js/main.js

```diff
@@ -0,0 +1,147 @@
+// TODO The hash changes, this is very brittle. See
+// https://github.com/thedodd/trunk/issues/230 or stop using trunk.
+import { dummyData } from "./dummy_data.js";
+import init, {
+  js_way_to_lanes,
+  js_lanes_to_tags,
+} from "./osm2lanes-npm/osm2lanes_npm.js";
+await init();
+
+// Setup the toolbox controls
+new Sortable(document.getElementById("toolbox"), {
```

Comment:
I'm using http://sortablejs.github.io/Sortable/ for the drag-and-drop functionality. It's pretty good so far, though I haven't figured out how to do some things with it. If anyone's used another library that I should look at, lmk